src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -95,6 +102,43 @@ public static void registerWorldGenerator(IWorldGenerator generator, int modGene
     }
 
     /**
+     * Registers a entity selector factory which is used to create predicates whenever a command containing selectors is executed
+     * Any non vanilla arguments that you expect has to be registered.
```

Comment:
I think it is important to point out *why* the arguments have to be registered. From what I understand, any argument that isn't added here will be ignored by the entity selector system.